app/serializers/post_serializer.rb

```diff
@@ -137,6 +137,11 @@ def duration
     object.duration ? object.duration.to_f : nil
   end
 
+  def comment_count
```

Comment:
This would be best in the post model itself. `user_visible_comment_count` or something. You can also drop the nullable check since this is never null. Not sure why I added that. There is also ``post_presenter.rb` which needs the same treatment

Reply:
Done and Done